Project.toml

```
@@ -0,0 +1,17 @@
+name = "CBOR"
+uuid = "7f3e1038-61bc-5414-967e-017c9d82adda"
+
+[deps]
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
+
+[compat]
+julia = "≥ 1.0.0"
```

@r2Fx I think this should be

```suggestion
julia = "1"
```

To cover all of Julia 1.0.0 to Julia 2.0.0, excluding upper bound.

The other packages are stdlibs, so I don't think we need (or can?) specify bounds.

---

Confirmed: https://discourse.julialang.org/t/announcement-automatic-merging-for-the-general-registry/29961/13